packages/components/warning-callout/template.njk

```diff
@@ -2,6 +2,9 @@
 <div class="nhsuk-warning-callout
 {%- if params.classes %} {{ params.classes }}{% endif %}"
 {%- for attribute, value in params.attributes %} {{attribute}}="{{value}}"{% endfor %}>
-  <h{{ headingLevel }} class="nhsuk-warning-callout__label">{{ params.heading | safe }}</h{{ headingLevel }}>
+  <h{{ headingLevel }} class="nhsuk-warning-callout__label">
+    {%- if params.heading != "Important" %}<span class="nhsuk-u-visually-hidden">Important: </span>{%- endif %}
```

Comment:
Suggest having the `<span role="text">` hack surrounding the visually hidden text and the heading text to deal with VoiceOver pausing and not reading the full heading, like we do for care cards: https://github.com/nhsuk/nhsuk-frontend/blob/eec78fc14fcb588487fc9d0f68c1312a78587002/packages/components/care-card/template.njk#L7

Reply:
Thanks @mcheung-nhs. I've applied the changes.

Reply:
👍